src/Directory.Packages.props

```diff
@@ -33,7 +33,7 @@
         <PackageVersion Include="OpenTelemetry.Instrumentation.Http" Version="1.9.0" />
         <PackageVersion Include="OpenTelemetry.Instrumentation.Runtime" Version="1.9.0" />
         <!--When updating Microsoft.Data.SqlClient, update license URL in scripts/notice-generation.ps1-->
```

Comment:
This comment says to update notice-generation script as well. Can you please fix that as well in the same PR?